### PR TITLE
Super helpful cleanup branch

### PR DESCRIPTION
**Pull request by JMD1986**
I just added some (what I thought would be) helpful stuff.

### CODE REVIEW

**Comment by mlgiardina** [comment]
:put_litter_in_its_place: :put_litter_in_its_place: :put_litter_in_its_place:

**Comment by jah2488** [comment]
👍🏻

**Comment by Graciexia** [comment]
:+1: nice

**Comment by BenHand** [comment]
:+1:  looks good

File: app/views/employees/index.html.erb [hunk 1]
```diff
@@ -10,15 +10,14 @@
   <div class="row">
     <p id="notice"><%= notice %></p>
 
-    <h1>All Employees</h1>
+    <h1><%= @company.name%> Employees</h1>
```
Comment (jah2488) [inline app/views/employees/index.html.erb]: Missing a space between `name` and `%>`

File: app/views/employees/show.html.erb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <div class="container">
+  <h2><%=@employee.company.name%> Employee</h2>
```
Comment (jah2488) [inline app/views/employees/show.html.erb]: Missing a space between `name` and `%>`

**Pull request merged**